### DISCUSSION TITLE
Use HTML for Source in Book Theme?

### DISCUSSION THREAD

**Discussion by eightbit** [executablebooks/meta#312] [Q&A]
Hi.
Book Theme is very elegant - nice job!
Is there a way to bypass the Markdown conversion and use HTML files as source?
Why? Because I have a lot of documentation to convert from MS Word and for this workflow it is easier to control and preview HTML than MarkDown - especially tables.
Thanks.

**Comment by rajk-cerebras** [comment]
Are you saying you already have a bunch of static html files and you want to bring them over into the book theme? I do that for some auto-generated API docs. I put them in source/_static/ folder, making sure that I have the index.html file as source/_static/index.html (or wherever under the _static folder). Then in some reStructuredText file I have the link like this: 

`My HTML Source <_static/myhtmldocs/html/index.html#http://>`. The `#http://` at the end is definitely a hack, I don't prefer it, but if I do not put it, it wasn't working. Alternately, if you have these html files served by http server somewhere, it is easier--you can just go: `My HTML Source <http://example.com/index.html>`. Was this what you were looking for?

**Reply by eightbit** [reply]
Thanks for your reply!
To be clear, what I want to achieve is to parse html the same way md is parsed. So in the example below, I want **source/_static/testZoom.html** to show up in the build with a link in the TOC which would then display the contents.

**Here's what I have, it's not parsing the html:**

**index.rst**
```
.. toctree::
   :caption: IABLE Docs
   :maxdepth: 4

   testMe01.md
   memberclicks.md
   addZoom.rst
```

**source/addZoom.rst**
```
addZoom
===========

Zoom Docs <_static/testZoom.html#http://>
```

When I open the link in the build, I get the contents of the addZoom .md, just as I do for the other md files, but it adds an extra / to the link to the html (`</ `rather than `<`).
### addZoom¶
Zoom Docs </_static/testZoom.html#http://>

**Reply by rajk-cerebras** [reply]
I see. So, maybe this helps. To place a link in the left sidebar to an already existing html, below are two scenarios:

1. Link to an external website, like you see the link to the GitHub repository in the left sidebar here: https://sphinx-book-theme.readthedocs.io/en/latest/index.html. To get this, just go:

```
.. toctree::
   :caption: IABLE Docs
   :maxdepth: 4

   testMe01.md
   memberclicks.md
   Zoom Docs <http:://example.com/testZoom.html>
```
This is not helpful to you, since you do not have the testZoom.html accessible at any `http://` location. This doesn't require any additional rst file to place your link. But the below option is applicable to you, as you use the `addZoom.rst` file to create the link.

2. Please try the below minor modification to your `source/addZoom.rst` file:

addZoom
===========

`Zoom Docs <_static/testZoom.html>`

As shown above, when we want to link to an HTML file, we have to put the whole thing in single quotation marks. This is the standard reStructuredText syntax.

**Reply by rajk-cerebras** [reply]
Well, the line above isn't showing the actual syntax, but it is 
```
`Zoom Docs <_static/testZoom.html>`
```

**Reply by eightbit** [reply]
Thanks again for the reply. I assume the tildes should be omitted, but I tried it both ways.
Still not working. The version w/the tildes prints in italic. 

In the build I still see the contents of source/addZoom.rst:
## addZoom
Test Print

Zoom Docs <_static/testZoom.html>

*Zoom Docs <_static/testZoom.html>*
